### ISSUE TITLE
Earth API: error 503

### ISSUE THREAD

**Issue by matfire** [nasa/api-docs#188]
Hello,

while trying to request earth images (using the Earth API) and a url structured as follows (`https://api.nasa.gov/planetary/earth/imagery?api_key=<key_here>&lat=4.1413&lon=-20.0094`), I get after a really long loading time the message: `upstream connect error or disconnect/reset before headers. retried and the latest reset reason: connection failure`.

Anyone else having this issue? Is there a way around it?

Stack: sveltekit

**Comment by STopalovv** [comment]
Hello,

Got the same issue using Vue and Typescript. Would be really thankful if someone got a workaround of some kind

**Comment by cjtorralba** [comment]
Yeah, seems like I am still getting this issue as of August, pretty disappointing. Would hope they get around to fixing it sometime

**Comment by navvye** [comment]
I am getting a similar error - I opened a similar error request.
